transform/transformers/PCKTransformer.py

```diff
@@ -117,11 +114,20 @@ def populate_period_data(self):
                 end_date = datetime.strptime(self.response['metadata']['ref_period_end_date'], "%Y-%m-%d")
                 self.data['12'] = end_date.strftime("%d/%m/%Y")
 
+    def round_currency_values(self):
+        """For RSI Surveys, round the values of the currency fields.
+        Rounds up if the value is .5
+        """
+        if self.survey['survey_id'] == '023':
```

Comment:
Need to start considering using constants or something for survey ids and things to make them easier to find/parse out in the code